utoipa-swagger-ui/src/lib.rs

```diff
@@ -388,6 +388,10 @@ impl<'a> From<Cow<'static, str>> for Url<'a> {
     }
 }
 
+pub const SWAGGER_STANDALONE_LAYOUT: &str = "StandaloneLayout";
+pub const SWAGGER_BASE_LAYOUT: &str = "BaseLayout";
+fn swagger_default_layout() -> String { SWAGGER_STANDALONE_LAYOUT.to_string() }
```

Comment:
I would try to use `&str` here since the are know values and thus no need to allocat them if just possible
```suggestion
fn swagger_default_layout() -> &'static str { SWAGGER_STANDALONE_LAYOUT }
```